CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [3.5.0] - 2023-05-03
 
+
```

Comment:
We can remove this here and create a new **Fixed** entry under the ** Unreleased** section describing we're fixing the refund event.
```suggestion
```